.evg.yml

```diff
@@ -327,7 +327,7 @@ tasks:
             cd stitch-java-sdk
             echo "running android tests"
             echo "test.stitch.baseURL=http://10.0.2.2:9090" >> local.properties
-            ./gradlew connectedAndroidTest --info --continue --warning-mode=all --stacktrace  < /dev/null
+            ./gradlew connectedAndroidTest jacocoTestReport --info --continue --warning-mode=all --stacktrace  < /dev/null
```

Comment:
[nit] this did not seem to work. let's get rid of this for now and file a ticket to fix android coverage

Reply:
looking into this a little more before removing